py/server/deephaven/jcompat.py

```diff
@@ -304,3 +304,32 @@ def _j_array_to_series(dtype: DType, j_array: jpy.JType, conv_null: bool) -> pd.
         s = pd.Series(data=np_array, copy=False)
 
     return s
+
+
+class SafeCloseable(JObjectWrapper):
```

Comment:
I was hesitant too, but ultimately decided SafeCloseable. The main difference between SafeCloseable and AutoCloseable is that SafeCloseable#close removes checked exception handling. Because there isn't checked exception handling in Python, I figure there's more symmetry with SafeCloseable.

Reply:
I saw that too, but I prefer to stay with SafeCloseable to match the Java one, even though we are really just using the AutoCloseable interface right now. It will not introduce any breaking changes in the future when we need to expose other methods in  SafeCloseable.

Reply:
Between:
* Chip's point of a more generalized AutoCloseable
* SafeCloseable not being as meaningful b/c of lacked checked exceptions in Python
* Most DH Python users prob aren't Java SafeCloseable users that might get confused at the difference,
I'll be changing into AutoCloseable.